pages/docs/concepts/asyncapi-document/adding-channels.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Adding Channels
+weight: 60
+---
+
+Adding [channels](../channel.md) in AsyncAPI specification allows organizing and categorizing messages, routing messages to the appropriate consumers, decoupling producers and consumers, scaling and extending the API, and providing clear documentation and communication about API's communication patterns. Additionally, alternative names for channels can be user-defined to suit specific context and preferences.
```

Comment:
Remember you should be reviewing your work with the preview links for EACH PR. If you had reviewed your document preview page, you would have noticed this link 404s. 

Pretty sure you just need to remove the `.md` from your link for it to work.

<img width="300" alt="Screen Shot 2023-08-31 at 7 17 20 PM" src="https://github.com/asyncapi/website/assets/19964402/6c1e5de3-8957-4394-a842-974a8fd74353">